core/src/main/scala/matryoshka/package.scala

```diff
@@ -262,6 +262,12 @@ package object matryoshka extends CofreeInstances with FreeInstances {
     */
   def once[A](f: A => Option[A]): A => A = expr => f(expr).getOrElse(expr)
 
+  /** Converts a failable fold into a non-failable, by returning the default
+   * upon failure.
+    */
+  def onceDefault[A, B](f: A => Option[B], default: B): A => B =
+    expr => f(expr).getOrElse(default)
+
```

Comment:
I like this, but it makes me think the `once` name above is wrong. I think this new one could be `orDefault` or `orElse`, while `once` could be renamed `orOriginal` or `orUnmodified`. And is there any reason to make the new one look like `(default: B)(f: A => Option[B])`?